builders/integrations/indexers/thegraph.md

```diff
@@ -15,15 +15,13 @@ The Graph is a decentralized and open-source indexing protocol for querying netw
 
 Furthermore, developers can build APIs, called Subgraphs. Users or other developers can use Subgraphs to query data specific to a set of smart contracts. Data is fetched with a standard GraphQL API. You can visit [their documentation](https://thegraph.com/docs/about/introduction#what-the-graph-is) to read more about The Graph protocol.
 
-With the introduction of Ethereum tracing modules in [Moonbase Alpha v7](https://github.com/PureStake/moonbeam/releases/tag/v0.7.0), The Graph is capable of indexing blockchain data in Moonbeam.
-
-This guide takes you through the creation of a simple subgraph for a Lottery contract on Moonbase Alpha.
+Due to the support of Ethereum tracing modules in Moonbeam, The Graph is capable of indexing blockchain data in Moonbeam. This guide takes you through the creation of a simple subgraph for a Lottery contract on Moonbase Alpha. This guide can be adapted for Moonriver.
```

Comment:
I would add that you don't need to run a node actually. (not sure if this is somewhere already). You can use The Graph hosted service as well